### ISSUE TITLE
[BUG] Subscriptions does not support CallbackCredentials, BasicAuth

### ISSUE THREAD

**Issue by ooglek** [Bandwidth/php-bandwidth-iris#85]
### Description
The `SubscriptionsModel.php` does not support `CallbackCredentials` for Basic Authentication nor PublicKey. When passing `CallbackCredentials` the Subscription is created, but without Authentication. 

Documented here: https://dev.bandwidth.com/docs/account/subscriptions/

### Sample Code Snippet
Using [PSYSH](https://psysh.org) and v3.3.1 of this SDK:

```php
// Succeeds in creating the Webhook WITHOUT Authentication
$a = $bw->account->subscriptions()->create(["OrderType" => "portins", "CallbackSubscription" => ["URL" => "https://my.website.example.com/bandwidth-webhook", "Expiry" => 3122064000], "CallbackCredentials" => [ "BasicAuthentication" => [ "Username" => "exampleuser", "Password" => "examplepass"] ] ]);

// No Authentication
$b = $bw->account->subscriptions()->subscription("2dc59c67-ac53-11ec-ba72-001e67e5b8b9");
/**
=> Iris\Subscription {#2776
     +"SubscriptionId": "2dc59c67-ac53-11ec-ba72-001e67e5b8b9",
     +"parent": Iris\Subscriptions {#2720
       +"parent": Iris\Account {#2717
         +"account_id": "1234567",
         +"subscriptions": Iris\Subscriptions {#2720},
       },
     },
     +"OrderType": "portins",
     +"CallbackSubscription": Iris\CallbackSubscription {#2748
       +"URL": "https://my.website.example.com/bandwidth-webhook",
       +"Expiry": "3122063131",
       +"Status": "200 OK - ok",
     },
   }
*/
```

### Actual Behavior
Subscription is created, but no `CallbackCredentials` are set, and no attempts are made to do Basic Authorization to the customer 
webhook endpoint.

### Expected Behavior
When specifying `CallbackCredentials` in the Subscriptions->create() method, the SDK should create the Webhook with the correct Credentials.

**Comment by ajrice6713** [comment]
Hey @ooglek - been a while, hope you are well. 

Thanks for raising up this issue - ive added it to our backlog for next sprint (starting 3/30) and we should be able to get this triaged and fix for a mid-april release!

**Comment by ajrice6713** [comment]
Hey @ooglek,

This is available in v3.4.0!

**Comment by ooglek** [comment]
Thanks AJ and Team! 😁
